templates/readme_md-template-v2.md

```diff
@@ -0,0 +1,128 @@
+*Note: this template is provided to give you a good idea about the kind of information you should include in a README.md file as a best practice. You may copy and edit it as you see fit for your community. What do you want people to know about your project who are just coming across it for the first time? OASIS Staff are happy to collaborate with you as well. Further README inspiration can be found [here](https://github.com/zalando/zalando-howto-open-source/blob/master/READMEtemplate.md#readme) and [here](https://github.com/hackergrrl/art-of-readme#readme).
+
+Table of contents
+-----------------
+
+* [Project Logo](#project-logo)
+* [Project Title](#project-title)
+* [Introduction](#introduction)
+* [Getting Started](#installation)
```

Comment:
I'm not sure, it depends on where this README is used. If it's in /OASIS-OPEN-PROJECT, then yes because it will be a more general introduction to the OP (maybe similar to what the [OpenC2 TC](https://github.com/oasis-tcs/openc2-tc-ops/blob/main/README.md) did. If this is used in a code repo, it will look more like this one.